root/static/scripts/common/components/Annotation.js

```diff
@@ -82,14 +84,18 @@ const Annotation = ({
                 date: formatUserDate($c.user, annotation.creation_date),
                 user: <EditorLink editor={annotation.editor} />,
               })}
+              {' '}
               {numberOfRevisions && numberOfRevisions > 1 ? (
                 <>
-                  {' '}
                   <a href={entityHref(entity, '/annotations')}>
                     {l('View annotation history')}
                   </a>
+                  {' / '}
```

Comment:
It appears we typically use ` | ` for separating edit links though, e.g. see `git grep -F "{' | '}" -- '*.js'`.